settings/types.go

```diff
@@ -60,3 +60,33 @@ type AirflowPools []struct {
 	PoolSlot        string `yaml:"slots"`
 	PoolDescription string `yaml:"description"`
 }
+
+// types for creating variables and connections yaml files
+
+type VariablesYAML struct {
```

Comment:
```suggestion
type DAGRunVariables struct {
```
I am just trying to figure out a better name for this since there are already two similar structs in the same file, it would be tough for a newcomer just to read the code and understand the use case of all of these structs if you have a better name then do go for it.

Reply:
sure